inc/codeception/config.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Altis Tests Config file.
+ *
+ * phpcs:disable PSR1.Files.SideEffects
+ *
+ * @package altis/dev-tools
+ */
+
+// Command is always executed from the project root on local servers.
+if ( ! defined( 'Altis\\PHPUNIT_PROJECT_ROOT' ) ) {
+	define( 'Altis\\PHPUNIT_PROJECT_ROOT', getcwd() );
+}
+
+defined( 'WP_TESTS_DOMAIN' ) or define( 'WP_TESTS_DOMAIN', 'example.org' );
+defined( 'WP_TESTS_EMAIL' ) or define( 'WP_TESTS_EMAIL', 'admin@example.org' );
+defined( 'WP_TESTS_TITLE' ) or define( 'WP_TESTS_TITLE', 'Test Blog' );
+
+// Ensure cache key salt is different for test runs.
+defined( 'WP_CACHE_KEY_SALT' ) or define( 'WP_CACHE_KEY_SALT', 'phpunit' );
+
+// Ensure tests use their own ElasticPress indexes.
+defined( 'EP_INDEX_PREFIX' ) or define( 'EP_INDEX_PREFIX', 'tests_' );
+
+// Set tests table prefix.
+$table_prefix = 'wp_';
+
+// Register shutdown event to clear the cache, also triggers if an error occurs.
+register_shutdown_function( function () {
+	if ( function_exists( 'wp_cache_flush' ) ) {
+		wp_cache_flush();
+	}
+} );
+
+function bootstrapCCWP() {
```

Comment:
Can't see where this gets called at all, is it a special / built in thing for wp-browser?

Reply:
It's being used by wp-browser as the bootstrap function, to run within the codeception bootstrapping process.

Reply:
Ah sorry - missed that! It needs a docblock then, and perhaps renaming to snake case.